ontology/yaml/resources/ELECTRICAL/entity_types/ABSTRACT.yaml

```diff
@@ -133,6 +152,16 @@ BTEPM:
   - battery_current_sensor
   - line_frequency_sensor  
 
+SBATM:
+  description: "Simple battery monitoringy typically used with UPS"
```

Comment:
This is not a valid abstract type, I would suggest adding fields as `opt_uses` to the existing `BATM` type.

Reply:
removed